README.rst

```diff
@@ -30,13 +30,13 @@ your system:
 
 .. code-block:: bash
 
-    $ wget http://get.sensiolabs.org/php-cs-fixer.phar -O php-cs-fixer
+    $ wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v1.11.4/php-cs-fixer.phar -O php-cs-fixer
```

Comment:
IMHO, you should keep it.

It will be useful for automatic install.

Like this for example: https://github.com/nexylan/docker-php-dev/blob/b07a75880a0bec07f40d06a04cb38bccb773d43c/7.0/Dockerfile#L72